ingestion_server/ingestion_server/cleanup.py

```diff
@@ -243,23 +252,25 @@ def _clean_data_worker(rows, temp_table, sources_config, all_fields: list[str]):
             if clean:
                 cleaned_data[update_field] = clean
                 log.debug(
-                    f"Updated {update_field} for {identifier} "
-                    f"from '{dirty_value}' to '{clean}'"
+                    f"Updated {update_field} for {identifier}\n\t"
+                    f"from '{dirty_value}' \n\tto '{clean}'"
                 )
         # Generate SQL update for all the fields we just cleaned
         update_field_expressions = []
         for field, clean_value in cleaned_data.items():
-            update_field_expressions.append(f"{field} = {clean_value}")
-            # Save cleaned values for later
-            # (except for tags, which take up too much space)
             if field == "tags":
+                update_field_expressions.append(f"{field} = {clean_value}")
                 continue
+            update_field_expressions.append(f"{field} = '{clean_value}'")
```

Comment:
The URL fields had single quotes added twice. As you can see above in the file (lines 117 and 119), I removed the quotes from the cleaned value so they won't appear in the file either (single quotes as the quoting character are problematic for later loading in the DB).

Reply:
I'm also confused about this :confused: 

Where were the duplicate quotes coming from, exactly? And why does the `clean_value` for tags not need to be quoted but everything else does?

Maybe avoiding the continue would help the clarity of this block:

```py
for field, clean_value in cleaned_data.items():
    if field != "tags":
        # Save cleaned values for later
        # (except for tags, which take up too much space)
        cleaned_values[field].append((identifier, clean_value))

    update_field_expressions.append(f"{field} = {clean_value}")
```

Never mind that I don't understand where the difference is coming from for the format of the string added to `update_field_expressions`, this version is a lot easier to understand, to me.

Reply:
> Where were the duplicate quotes coming from, exactly?

The cleaned values have quotes added in the return of the cleaning function:
https://github.com/WordPress/openverse/blob/d562f4719aefdf7d90c13ca6738a5e8c27ce352f/ingestion_server/ingestion_server/cleanup.py#L108-L111

>  And why does the clean_value for tags not need to be quoted but everything else does?

The `psycopg2.extras.Json` function for tags already adapts a Python object to :sql:`json` data type, so they don't need extra quotes wrapping.

The difference in the string added to `update_field_expressions` is the quotes. I assume this was previously done that way to avoid this confusion, but it's necessary to have the cleaned values without these quotes in the file so they don't interfere with the copy upload to a DB table later.

An alternative would be to perform more string manipulation to remove the quotes before saving the rows in files, but this seemed unnecessary and more prone to error to me.

Reply:
I agree with Staci, but I also don't want to block on it because of this code being deleted soon anyway :shrug:

Reply:
I updated the comment. If it's still not clear, it's open to suggestions. I didn't think it would bring so much confusion 😅